### PR TITLE
(Frontend) AHR Black History Month

### LINKED ISSUE #1423: add additional topics from UHC - target towards black history month
Link to Google doc discussing options with team: https://docs.google.com/document/d/1jkYiLN8teOmcKU1HdJXMK5Cx0pFkUDLM4a1UonYjbrg/edit

### PR DESCRIPTION
**Pull request by benhammondmusic**
## Preview Link

<a href="https://deploy-preview-1437--health-equity-tracker.netlify.app/exploredata?mls=1.asthma-3.00" target="_blank">
  <br />
Click to demo updates </a>   

## To Do
- [ ] REMOVE temp files / env vars
- [x] PREVENTABLE HOSP  "Adults ages 65 and older, who have a rate of preventable hospitalization more than 12 times that of those ages 18-44." we don't have any age brackets available, and no BY AGE charts are represented on the AHR site. It appears they are drawing this insight from the original source (https://www.hcup-us.ahrq.gov/reports/statbriefs/sb259-Potentially-Preventable-Hospitalizations-2017.jsp) without importing the age brackets into their data set. 
- [x] VOTER we are only looking at the subset for `presidential`; it looks like they average in the non-demographic breakdown `midterm` data as well for their "all" metric. should do this somehow? 
- [x] VOTER why is there no data in AHR csv for 65+ ?? Emailed AHR on feb 8th
- [x] CHRONIC KIDNEY (AHR reports "Black and Hispanic adults compared with white and Asian adults" but our report AND the chart by RACE on AHR both show black and AIAN as highest, and hispanic and asian as lowest". Looking back a few years Hispanic was 2nd highest so I would guess their analysis hasn't been updated. Confirm this assumption
- [x] fix frontend tests for BRFSS provider
- [x] fix missing age buckets for voter: data table is ONLY showing `18-24`;
- [x] fix missing age buckets for voter: data table is missing data and population data for some age buckets. issue is on the backend as the /tmp json files are missing the data to begin with
- [x] fix missing age buckets for voter: simple bar chart is MISSING `18-24` 
- [x] fix missing age buckets for voter: disparity bar chart is MISSING `18-24`
- [x] PREVENTABLE HOSP "Black adults compared with Asian/Pacific Islander adults. " Asian is missing and other variables because of mismatched racial groups. 
- [x] update data sources / meta data. 
- [x] Confirm which data is actually sourced originally from BRFSS survey and which data might come from other surveys (voting?)

### Spot check our results against AHR
- [x] Asthma
- [x] Cardiovascular disease
- [x] Chronic Kidney Disease 
- [x] Avoided Care due to Costs
- [x] Preventable Hospitalization
- [x] Voter Participation

## Description

Adds several new determinants to the tracker, to balance the number of determinants per category and to highlight some determinants that particularly affect Black Americans (with the focus of Feb being Black History Month)

### Chronic Disease
- Asthma: Percentage of adults who reported being told by a health professional that they currently have asthma
- Cardiovascular diseases: Percentage of adults who reported being told by a health professional that they had angina or coronary heart disease; a heart attack or myocardial infarction; or a stroke
- Chronic Kidney Disease: Percentage of adults who reported being told by a health professional that they have kidney disease not including kidney stones, bladder infection or incontinence

### Social / Political Determinant
- Avoided Care due to Cost: Percentage of adults who reported a time in the past 12 months when they needed to see a doctor but could not because of cost
- Preventable Hospitalizations: Discharges following hospitalization for diabetes with short- or long-term complications, uncontrolled diabetes without complications, diabetes with lower-extremity amputation, chronic obstructive pulmonary disease, angina without a procedure, asthma, hypertension, heart failure, dehydration, bacterial pneumonia or urinary tract infection per 100,000 Medicare beneficiaries ages 18 and older continuously enrolled in Medicare fee-for-service Part A
- Voter Participation (Presidential): Percentage of U.S. citizens ages 18 and older who voted in the last presidential national election

## Motivation and Context
closes #1423 

## Screenshots (if appropriate):
Screen Shot 2022-02-04 at 3 18 52 PM

## Types of changes

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by benhammondmusic** [comment]
We now use either midterm, presidential, or the average of both. results match AHR's average reporting for state totals

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx [hunk 1]
```diff
@@ -264,7 +267,7 @@ function MethodologyTab() {
                   <li>
                     BRFSS data broken down by race and ethnicity is not
                     available at the county level, so the tracker does not
-                    display diabetes or COPD data at the county level either.
```
Comment (joshzarrabi) [inline frontend/src/pages/DataCatalog/MethodologyTab.tsx:267]: good catch

File: frontend/src/data/variables/BrfssProvider.ts [hunk 2]
```diff
@@ -128,6 +150,33 @@ class BrfssProvider extends VariableProvider {
           row.frequent_mental_distress_per_100k,
           row.population
         ),
+      estimated_total_preventable_hospitalizations: (row) =>
```
Comment (joshzarrabi) [inline frontend/src/data/variables/BrfssProvider.ts:153]: @benhammondmusic Can we add a todo to delete these

**Review by joshzarrabi — APPROVED** [review]

File: frontend/.env.deploy_preview [hunk 3]
```diff
@@ -2,3 +2,5 @@ REACT_APP_DEPLOY_CONTEXT=deploy_preview
 REACT_APP_BASE_API_URL=https://dev.healthequitytracker.org
 
 REACT_APP_FORCE_STATIC=geographies.json
+REACT_APP_FORCE_STATIC=geographies.json,uhc_data-race_and_ethnicity.json,uhc_data-age.json,uhc_data-sex.json,acs_population-by_age_state.json
```
Comment (joshzarrabi) [inline frontend/.env.deploy_preview:5]: delete this before merging

**Pull request merged**